.travis/build-and-test.bash

```diff
@@ -10,11 +10,9 @@ function build_and_test_ubuntu() {
     popd
     pushd /build
     export CXXFLAGS="-O2 -g"
-    /src/configure \
-        --enable-pmc \
-        --enable-python \
-        --prefix=/usr
-    make distcheck -j2 DISTCHECK_CONFIGURE_FLAGS="--enable-pmc --enable-python --prefix=/usr" VERBOSE=1
+    CONFIGURE_FLAGS="--enable-pmc --enable-python --prefix=/usr"
+    /src/configure ${CONFIGURE_FLAGS}
+    make distcheck -j2 DISTCHECK_CONFIGURE_FLAGS=${CONFIGURE_FLAGS} VERBOSE=1
```

Comment:
In bash, ```foo=bar baz``` works to assign a variable. However, when invoking a command white space remain white spaces. You need to wrap ```${CONFIGURE_FLAGS}``` in quotation marks in the ```make distcheck``` line.